hw/ip/clkmgr/doc/_index.md

```diff
@@ -154,6 +154,23 @@ The following is a high level block diagram of the clock manager.
 
 ![Clock Manager Block Diagram](clkmgr_block_diagram.svg)
 
+### Reset Domains
```

Comment:
@matutem let me know if this makes sense to you.

Reply:
Yes, thanks.